src/classrooms/classrooms.service.ts

```diff
@@ -0,0 +1,27 @@
+import { Injectable } from "@nestjs/common";
+
+import { EntityManager } from "@mikro-orm/postgresql";
+
+import { UsersRepository } from "@/users/users.repository";
+
+import { CreateClassroomDto } from "./classrooms.dto";
+import { ClassroomsRepository } from "./classrooms.repository";
+
+@Injectable()
+export class ClassroomsService {
+  constructor(
+    private readonly classroomRepository: ClassroomsRepository,
+    private readonly userRepository: UsersRepository,
+    private readonly em: EntityManager,
+  ) {}
+
+  async createClassroom(classroomDto: CreateClassroomDto, userId: number) {
+    const user = await this.userRepository.findOneOrFail({ id: userId });
+    const classroom = this.classroomRepository.create({
+      ...classroomDto,
+      teacher: user.teacher,
+    });
+    await this.em.persistAndFlush(classroom);
+    return classroom;
```

Comment:
```suggestion
    const user = await this.userRepository.findOneOrFail({ id: userId });

    const classroom = this.classroomRepository.create({
      ...classroomDto,
      teacher: user.teacher,
    });

    await this.em.persistAndFlush(classroom);

    return classroom;
```

Let's add line breaks here